examples/webgl_materials_texture_3d.html

```diff
@@ -0,0 +1,213 @@
+	precision highp sampler3D;
+
+	uniform sampler3D diffuse;
+	in vec3 vUv;
+	out vec4 out_FragColor;
+
+	void main() {
+
+		vec4 color = texture( diffuse, vUv );
+		color.r *= 1.5; // lighten a bit
+		color.gb = vec2( color.r );
+
+		out_FragColor = vec4( color.rgb, 1.0 );
+
+	}
+	</script>
+	<body>
+		<div id="info">
+			<a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> - 3D Texture<br />
+			Scanned head data by
+			<a href="https://www.codeproject.com/Articles/352270/Getting-started-with-Volume-Rendering" target="_blank" rel="noopener">Divine Augustine</a><br />
+			licensed under
+			<a href="https://www.codeproject.com/info/cpol10.aspx" target="_blank" rel="noopener">CPOL</a>
+		</div>
+
+		<script src="../build/three.js"></script>
+		<!--<script src="js/controls/OrbitControls.js"></script>-->
+
+		<script src="js/libs/jszip.min.js"></script>
+
+		<script src="js/Detector.js"></script>
+		<script src="js/libs/stats.min.js"></script>
+
+		<script>
+
+			if ( ! Detector.webgl ) Detector.addGetWebGLMessage();
+
+			var camera, scene, mesh, renderer, stats;
+
+			var planeWidth = 50;
+			var planeHeight = 50;
+
+			var depthStep = 1 / ( 109 * 3 );
+
+			init();
+			animate();
+
+			function init() {
+
+				var container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 0.1, 2000 );
+				camera.position.z = 70;
+
+				scene = new THREE.Scene();
+
+				// width 256, height 256, depth 109, 8-bit, zip archived raw data
+
+				new THREE.FileLoader()
+					.setResponseType( 'arraybuffer' )
+					.load( 'textures/3d/head256x256x109.zip', function ( data ) {
+
+					var zip = new JSZip( data );
+					var array = zip.files[ 'head256x256x109' ].asUint8Array();
+
+					var texture = new THREE.Texture3D( array, 256, 256, 109 );
+
+					texture.format = THREE.RedFormat;
+					texture.type = THREE.UnsignedByteType;
+
+					texture.needsUpdate = true;
+
+					var material = new THREE.ShaderMaterial( {
+						uniforms: {
+							diffuse: { value: texture },
+							depth: { value: 0 },
+							size: { value: new THREE.Vector2( planeWidth, planeHeight ) }
+						},
+						vertexShader: document.getElementById( 'vs' ).textContent.trim(),
+						fragmentShader: document.getElementById( 'fs' ).textContent.trim()
+					} );
+
+					var geometry = new THREE.PlaneBufferGeometry( planeWidth, planeHeight );
+
+					mesh = new THREE.Mesh( geometry, material );
+
+					scene.add( mesh );
+
+				} );
+
+				// var controls = new THREE.OrbitControls( camera );
+
+				// 3D Texture is available on WebGL 2.0
+
+				var canvas = document.createElement( 'canvas' );
+				var context = canvas.getContext( 'webgl2' );
+
+				renderer = new THREE.WebGLRenderer( { antialias: true, canvas: canvas, context: context } );
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
```

Comment:
You can probably omit this line because its also in `onWindowResize` which you call just below,

Reply:
This line and the previous one are typical for all `three.js` examples. I would not change them.